Modules/Devices/src/Devices.Infrastructure/PushNotifications/DirectPush/ApplePushNotificationService/ApnsMessageBuilder.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Net;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Enmeshed.Tooling.Extensions;
+
+namespace Backbone.Modules.Devices.Infrastructure.PushNotifications.DirectPush.ApplePushNotificationService;
+
+public class ApnsMessageBuilder
+{
+    private readonly Payload _notification = new();
+    private readonly JsonSerializerOptions _jsonSerializerOptions = new() { Converters = { new DateTimeConverter() }, PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
+    private readonly HttpRequestMessage _request;
+
+    public ApnsMessageBuilder(string appBundleIdentifier, string path, string jwt)
+    {
+        _request = new HttpRequestMessage(HttpMethod.Post, new Uri(path))
+        {
+            Version = HttpVersion.Version30,
+            Headers =
+            {
+                {"apns-topic", appBundleIdentifier},
+                {"apns-expiration", "0"},
+                {"apns-priority", "5"},
+                {"apns-push-type", "background"}
+            }
```

Comment:
I changed the HTTP version to HTTP/3 and the apns-push-type to "background". Can you please test if it still works? And if it doesn't, undo whatever causes the error?